app/assets/javascripts/discourse/app/routes/about.js

```diff
@@ -1,19 +1,32 @@
 import { ajax } from "discourse/lib/ajax";
+import Category from "discourse/models/category";
+import Site from "discourse/models/site";
 import DiscourseRoute from "discourse/routes/discourse";
 import I18n from "discourse-i18n";
 
 export default DiscourseRoute.extend({
-  model() {
-    return ajax("/about.json").then((result) => {
-      const yearAgo = moment().locale("en").utc().subtract(1, "year");
-      result.about.admins = result.about.admins.filter(
-        (r) => moment(r.last_seen_at) > yearAgo
-      );
-      result.about.moderators = result.about.moderators.filter(
-        (r) => moment(r.last_seen_at) > yearAgo
-      );
-      return result.about;
+  async model() {
+    const result = await ajax("/about.json");
+    const users = Object.fromEntries(
+      result.users.map((user) => [user.id, user])
+    );
+    result.categories?.forEach((category) => {
+      Site.current().updateCategory(category);
     });
+
+    const yearAgo = moment().locale("en").utc().subtract(1, "year");
```

Comment:
locale isn't needed here, is it?

```suggestion
    const yearAgo = moment().utc().subtract(1, "year");
```

Reply:
Not sure because I just moved this around. However, I see a lot of similar usage. Is it wrong?

Reply:
Not "wrong" - just doesn't (shouldn't 😄) do anything if you don't print out the date